### ISSUE TITLE
GL Selection Actions/Reducer

### ISSUE THREAD

**Issue by klappy** [unfoldingWord/translationCore#3858]
The GL Selection UI was created but not the actions/reducer.

To be able to complete the GL Selection we need a reducer to hold the selected GL and actions to populate it.

Needs to update hardcoded `en` in `ProjectDataLoadingActions.js:29` that dynamically loads the GroupIndex aka Menu Titles from the GL.

Needs to update the tW Article loading from the GL's copy of tW articles in the user resources path. Via `ResourceActions.js:134` update `en` to selected GL from new reducer.

**Comment by klappy** [comment]
@benjore @jag3773: After working with @richmahn on using the selected GL for the tW tool, we realized that only the UI components were created for the GL selection and this story is needed to make it usable.

**Comment by richmahn** [comment]
- [ ] [2] Hook up GL selection to save the GL to state
- [ ] [5] Create ProjectSettings reducer to contain the GL of each tool of the currently selected project
- [ ] [3] Have the GL for Project/Tool save and load from file to remember the user's selection of each Project/tool
- [ ] [2] Write test for changing the GL and saving
- [ ] [2] Write test for loading a project/tools language
- [ ] [2] Write test for ProjectSettings reducer
- [ ] [2] PR for GL Selection Actions/Reducer

**Comment by richmahn** [comment]
@cckozie What has been updated to use the language selected from a tools Gateway language selector is as follows:

Changing the language of a tool of the open project (e.g. Hindi) and then leaving that page and going back (but NOT selecting a new project) will continue to remember the selected language in the tool card's GL selector. However, if you open a new project or close/open the app, it will NOT persist. There is another story to persist that data.

Going into tW should use that Gateway language for all the places tW helps is accessed: left sidebar, middle check info pane, right tHelps article display. This is NOT for use with the Verse Check pane as the noted word that the user is to select will come from the aligned Bible of the gateway language, not the tHelps.

So in short, this just currently just tells tW what language to display the title and body of a tW article. No Bible/Scripture implementation nor save of selected GL for a project/tool.

**Comment by cckozie** [comment]
I am seeing 3 issues when testing this although I'm not sure if they are problems created herein or if they are data issues or problems somewhere else.
In the following images note:
1. Multiple # signs in the hindi text
2. Lack of font styling on the helps title in the right sidebar
3. According to Google translate, the wrong help is being displayed
![en](https://user-images.githubusercontent.com/8171759/36740045-f969c4b8-1ba6-11e8-80f0-c2c9d21693ac.png)
![hi](https://user-images.githubusercontent.com/8171759/36740210-66b05fdc-1ba7-11e8-8652-6271501cdca7.png)

![gt](https://user-images.githubusercontent.com/8171759/36740084-0e7877e6-1ba7-11e8-9f8b-2e1aa1dac4d5.png)

**Comment by cckozie** [comment]
All of the noted issues are data issues. The only thing to be tested is whether or not the dropdown works, and that does.
